pkgs/development/tools/gdlv/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "gdlv";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "aarzilli";
+    repo = "${pname}";
```

Comment:
```suggestion
    repo = pname;
```

or better

```suggestion
    repo = "gdlv";
```

Reply:
going with the latter, thanks for the review!